src/notification-renderer.js

```diff
@@ -31,7 +32,13 @@ export class NotificationRenderer {
     notificationController.slot.add(notificationController.view);
 
     notificationController.showNotification = () => {
+            
       this.notificationControllers.push(notificationController);
+     
+      if (this.notificationControllers.length >= (globalSettings.limit + 1)) {
```

Comment:
Can you switch from globalSettings.limit to settings.limit? It is configurable then.